### PR TITLE
Fix for Adjust event . See: #2595

### PR DESCRIPTION
**Pull request by cnevinc**
There are three commits in this PR.

The first one removes redundant source set in build.gradle.

The second one fixes the problem that Adjust didn't track any referral due to the mis-configure Adjust environment. The root cause of this bug is because I should use the value from `com.adjust.sdk.AdjustConfig.ENVIRONMENT_PRODUCTION` directly. instead of `"com.adjust.sdk.AdjustConfig.ENVIRONMENT_PRODUCTION"`

The third commit adds two missing Adjust event.

### CODE REVIEW

**Comment by mTwTm** [comment]
Fixes #2992. @cnevinc do you have access to the sandboxed Adjust to verify there is no other problem?

**Comment by cnevinc** [comment]
> Fixes #2992. @cnevinc do you have access to the sandboxed Adjust to verify there is no other problem?

I'm trying to do this with @elin-moco now

File: app/src/main/java/org/mozilla/focus/activity/MainActivity.java [hunk 1]
```diff
@@ -551,6 +551,7 @@ public void onMenuItemClicked(View v) {
                 Intent intent = new Intent(this, PrivateModeActivity.class);
                 startActivity(intent);
                 overridePendingTransition(R.anim.tab_transition_fade_in, R.anim.tab_transition_fade_out);
+                TelemetryWrapper.togglePrivateMode(true);
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: Consider giving this a version so we know why the volume suddenly increases.

Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: added

Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/activity/MainActivity.java]: I think maybe it should be a new ping. SInce it's from Tab Tray

**Pull request merged**